modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_0_0/PollsUpgradeProcess.java

```diff
@@ -0,0 +1,1074 @@
+					"structureVersionId, name, description, settings_, ",
+					"version, status, statusByUserId, statusByUserName, ",
+					"statusDate) values (0, 0, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ",
+					"?, ?, ?, ?, ?, ?)"))) {
+
+			preparedStatement.setLong(1, increment());
+			preparedStatement.setLong(2, groupId);
+			preparedStatement.setLong(3, companyId);
+			preparedStatement.setLong(4, userId);
+			preparedStatement.setString(5, userName);
+			preparedStatement.setTimestamp(6, createDate);
+			preparedStatement.setLong(7, formInstanceId);
+			preparedStatement.setLong(8, structureVersionId);
+			preparedStatement.setString(9, name);
+			preparedStatement.setString(10, description);
+			preparedStatement.setString(11, StringPool.BLANK);
+			preparedStatement.setString(
+				12, DDMFormInstanceConstants.VERSION_DEFAULT);
+			preparedStatement.setInt(13, WorkflowConstants.STATUS_APPROVED);
+			preparedStatement.setLong(14, userId);
+			preparedStatement.setString(15, userName);
+			preparedStatement.setTimestamp(16, statusDate);
+
+			preparedStatement.executeUpdate();
+		}
+	}
+
+	private long _addDDMStorageLink(
+			long companyId, long structureId, long structureVersionId)
+		throws Exception {
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				StringBundler.concat(
+					"insert into DDMStorageLink (mvccVersion, ctCollectionId, ",
+					"uuid_, storageLinkId, companyId, classNameId, classPK, ",
+					"structureId, structureVersionId) values (0, 0, ?, ?, ?, ",
+					"?, ?, ?, ?)"))) {
+
+			preparedStatement.setString(1, PortalUUIDUtil.generate());
+			preparedStatement.setLong(2, increment());
+			preparedStatement.setLong(3, companyId);
+			preparedStatement.setLong(
+				4, PortalUtil.getClassNameId(DDMContent.class));
+
+			long storageId = increment();
+
+			preparedStatement.setLong(5, storageId);
+
+			preparedStatement.setLong(6, structureId);
+			preparedStatement.setLong(7, structureVersionId);
+
+			preparedStatement.executeUpdate();
+
+			return storageId;
+		}
+	}
+
+	private long _addDDMStructure(
+			long groupId, long companyId, long userId, String userName,
+			Timestamp createDate, Timestamp modifiedDate, String name,
+			String description, String definition, Timestamp lastPublishDate)
+		throws Exception {
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				StringBundler.concat(
+					"insert into DDMStructure (mvccVersion, ctCollectionId, ",
+					"uuid_, structureId, groupId, companyId, userId, ",
+					"userName, versionUserId, versionUserName, createDate, ",
+					"modifiedDate, parentStructureId, classNameId, ",
+					"structureKey, version, name, description, definition, ",
+					"storageType, type_, lastPublishDate) values (0, 0, ?, ?, ",
+					"?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"))) {
+
+			preparedStatement.setString(1, PortalUUIDUtil.generate());
+
+			long structureId = increment();
+
+			preparedStatement.setLong(2, structureId);
+
```

Comment:
@natocesarrego I think it's necessary because I'm initializing the variable `long structureId` above and the`preparedStatement.setLong(2, structureId);` depends on this variable.

I was also following the example from [DLFileEntryTypeDataDefinitionIdUpgradeProcess.java#L223-L235](https://github.com/liferay/liferay-portal/blob/08c6b85908e5745c20b53fd5c8e3032bbf15e165/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_0/DLFileEntryTypeDataDefinitionIdUpgradeProcess.java#L223-L235).